src/common/components/page-links/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react';
+
+import styles from './styles.css';
+
+const LINK_LABEL_OPTIONS = ['first', 'prev', 'next', 'last'];
+
+export default class PageLinks extends Component {
+
+  render() {
+    const renderLink = this.renderLink.bind(this);
+
+    let links = LINK_LABEL_OPTIONS.map((item) => {
+      return renderLink(
+        this.props[item],
+        item
+      );
+    });
+
+    return (
+      <ul className={ styles.container }>
+        { links }
+      </ul>
+    );
+  }
+
+  renderLink(item, label) {
+    let url;
+
+    if (typeof item == 'string') {
```

Comment:
I add comments to my PRs purely to make reviewing easier, but if you think a comment is needed in the source code I can add one 👍